src/doc/rustc/src/platform-support/x86_64-unknown-linux-none.md

```diff
@@ -14,6 +14,11 @@ This target is cross compiled and can be built from any host.
 
 This target has no support for host tools, std, or alloc.
 
+One of the primary motivations of the target is to write a dynamic linker and libc in Rust.
+For that, the target defaults to position-independent code and position-independent executables (PIE) by default.
+To be able to run a PIE, you need a dynamic linker.
```

Comment:
I don't think this is quite true, since C programs compiled with `-static-pie` work with no dynamic linker. What about something like this instead?
```suggestion
PIE binaries need relocation at runtime. This is usually done by the dynamic linker or libc.
```